search/tests/mock_search_engine.py

```diff
@@ -281,35 +287,22 @@ def load_index(cls, index_name):
         """ load the index, if necessary from the backed file """
         cls._load_from_file()
         if index_name not in cls._mock_elastic:
-            cls._mock_elastic[index_name] = {}
+            cls._mock_elastic[index_name] = []
             cls._write_to_file()
 
         return cls._mock_elastic[index_name]
 
     @classmethod
-    def load_doc_type(cls, index_name, doc_type):
-        """ load the documents of type doc_type, if necessary loading from the backed file """
-        index = cls.load_index(index_name)
-        if doc_type not in index:
-            index[doc_type] = []
-            cls._write_to_file()
-
-        return index[doc_type]
-
-    @classmethod
-    def add_documents(cls, index_name, doc_type, sources):
-        """ add documents of specific type to index """
-        cls.load_doc_type(index_name, doc_type).extend(sources)
+    def add_documents(cls, index_name, sources):
+        """ add documents to index """
```

Comment:
formatting. Do not forget and go through.

Reply:
Done